apps/datahub/src/app/record/navigation-bar/navigation-bar.component.html

```diff
@@ -10,15 +10,10 @@
     "
   >
     <div
-      class="h-[60px] flex min-w-[60px] items-center justify-center lg:justify-end lg:pr-4"
+      class="h-[60px] flex min-w-[60px] items-center justify-center lg:justify-end lg:pr-4 text-sm"
       [class]="displayMobileMenu ? 'border-r-2 border-gray-300' : ''"
     >
-      <gn-ui-button
-        data-cy="backButton"
-        (buttonClick)="back()"
-        type="outline"
-        [style.--gn-ui-button-font-size]="'24px'"
-      >
```

Comment:
Thanks for this change, looks cleaner indeed! So do I understand correctly that the icons always need to be in `text-sm` and then the size `1.5em` from the datahub `app.module` is applied on it?

Reply:
Yes, it should always be like this now, I tried to homogenize.